include/gul14/to_number.h

```diff
@@ -488,17 +488,19 @@ inline optional<NumberType> strtold_wrapper(gul14::string_view str) noexcept
  */
 // Overload for unsigned integer types.
 template <typename NumberType,
-          std::enable_if_t<std::is_integral<NumberType>::value &&
-                           std::is_unsigned<NumberType>::value, int> = 0>
+          std::enable_if_t<std::is_integral<NumberType>::value
+                           and std::is_unsigned<NumberType>::value
+                           and not std::is_same<NumberType, bool>::value, int> = 0>
 constexpr inline optional<NumberType> to_number(gul14::string_view str) noexcept
 {
     return detail::to_unsigned_integer<NumberType>(str);
 }
 
 // Overload for signed integer types.
 template <typename NumberType,
-          std::enable_if_t<std::is_integral<NumberType>::value &&
-                           std::is_signed<NumberType>::value, int> = 0>
+          std::enable_if_t<std::is_integral<NumberType>::value
+                           and std::is_signed<NumberType>::value
+                           and not std::is_same<NumberType, bool>::value, int> = 0>
```

Comment:
The test for bool here is unnecessary because `is_signed<bool> == false` (that said, it does not really hurt either).

Reply:
Fixed by dropping the change.